### DISCUSSION TITLE
unminified version

### DISCUSSION THREAD

**Discussion by hamburger123456** [AntoineW/luge#43] [General]
Hi, 
I like your code.
Is there any possibility to get a compiled but unminified Version of luge.js?
I do not able to use serversite scripting and would like to see what happens in your code.
thx in advance.
joe

**Comment by AntoineW** [comment]
Hi,
Feel free to fork the repo and update the gulp file to change the rollup config. You should be able to output a plain file by removing the terser plugin.

**Comment by hamburger123456** [comment]
Thx for your response.
To fork the repro (and compile a new version) is difficult for someone who not use notes and npm package. It will be great for other useres to have a plain-version by author.
